index.d.ts

```diff
@@ -16,6 +16,6 @@ stripIndent(string);
 //	cake
 ```
 */
-declare function stripIndent(string: string): string;
+declare function stripIndent(string: string, testString?: string): string;
```

Comment:
It should be an options-object. Example: https://github.com/sindresorhus/typescript-definition-style-guide#documentation

Can you suggest some other option names than `testString`? I'm not 100% sold on that name.

Reply:
Thanks for a quick reply! Will address both issues soon.

How about `indentString`?

Reply:
`indentString` is worse. It makes it sound like it's the indent string to strip.

Reply:
Fully agree, finally thought about `parseString`. Anyway, closing.